entrypoint.sh

```diff
@@ -1044,6 +1047,10 @@ function load_nfsrdma() {
         exec_cmd "modprobe nvme-rdma"
         exec_cmd "modprobe rpcrdma"
     fi
+
+    if [[ "${ENABLE_NFSRDMA_NO_NVME}" = true ]] ; then
+        exec_cmd "modprobe rpcrdma"
```

Comment:
we want to load this regardless of if nvme or no nvme.  this bit is related to nfs no ?